src/openapi/streaming/subscription.js

```diff
@@ -372,23 +374,34 @@ function setState(state) {
 function Subscription(streamingContextId, transport, serviceGroup, url, subscriptionArgs, onSubscriptionCreated, onUpdate, onError) {
 
     /**
-     * The streaming context id identifies the particular streaming connection that this subscription will use
+     * The streaming context id identifies the particular streaming connection that this subscription will use.
      * @type {string}
      */
     this.streamingContextId = streamingContextId;
 
     /**
-     * The reference id is used to identify this subscription
+     * The reference id is used to identify this subscription.
      * @type {string}
      */
     this.referenceId = null;
 
     /**
-     * The action queue
+     * The action queue.
      * @type {SubscriptionQueue}
      */
     this.queue = new SubscriptionQueue();
 
+    /**
+     * The subscription format. Used to derive serialization method for incoming subscription messages.
+     * @type {String}
+     */
+    this.format = subscriptionArgs.Format;

```

Comment:
how do we stop people forgetting a Format argument? maybe we can wrap streaming and always provide it?